health-services/project-factory/src/server/utils/campaignUtils.ts

```
@@ -2361,23 +2361,37 @@ async function createProject(
-      let targetNo = parseInt(codesTargetMapping?.[boundaryCode][key]);
-      let beneficiaryType = key;
-      targets.push({
-        beneficiaryType: beneficiaryType,
-        targetNo: targetNo,
-      });
+    let targets: any[] = [];
+    const alreadyPresentTargets = project?.targets || [];
+
+    // Iterate through the mappings and update/create targets
+    for (const beneficiaryType in codesTargetMapping[boundaryCode]) {
+      const targetNo = parseInt(codesTargetMapping[boundaryCode][beneficiaryType], 10); // Ensure numeric conversion
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)